actionview/lib/action_view/helpers/tag_helper.rb

```diff
@@ -26,7 +26,100 @@ module TagHelper
       PRE_CONTENT_STRINGS[:textarea]  = "\n"
       PRE_CONTENT_STRINGS["textarea"] = "\n"
 
+      class TagBuilder #:nodoc:
+        include TagHelper
 
+        VOID_ELEMENTS = %i(base br col embed hr img input keygen link meta
+        param source track wbr).to_set
+
+        def initialize(view_context)
+          @view_context = view_context
+        end
+
+        private
+          def tag_string(name, content = nil, escape_attributes: true, **options, &block)
+            content = @view_context.capture(self, &block) if block_given?
```

Comment:
That's not the same kind of newline. It's perfectly fine to separate parts of a method :)